Fixes #1

richardstrnad opened the pull request:
This should fix the issue we faced

richardstrnad committed:
Unified spaces/tabs

richardstrnad committed:
Fixed #1

sc211166 commented:
Hello ,
I wonder if there's a way to modify the script to import MAC HOSTS thanks

AKA-47 committed:
Fixed Password: passwordform="plain"

<Username>admin</Username : Username should be of valid administrator profile user
<Password passwordform="plain">  : When this method is used password needs to provide in plain text
<Password passwordform="encrypt"> : when this method is used password should be in encrypted text derive from user property.

AKA-47 committed:
fixed missing <Password

richardstrnad committed:
Merge pull request #1 from AKA-47/master

Fixed Password: passwordform="plain"
